oci-library/oci-hol/java-management/workshops/freetier/manifest.json

```diff
@@ -42,6 +42,11 @@
             "title": "Lab 7: Track Java Usage in a Container",
             "filename":"../../track-java-usage-in-a-container/track-java-usage-in-a-container.md"
             
+        },
+        {
+            "title": "Lab 8: Set up logrotate",
```

Comment:
"Set up logrotate" to be changed to "Set Up Logrotate" as per Self QA instructions Section 4 (Lab Section)